### PR TITLE
chore: updates golden sig+ contributor file fixes

### PR DESCRIPTION
**Pull request by jpvajda**
This PR updates the golden signals dashboard image to follow the correct naming convention as well as fixes the use of packs in the contributor file.

### CODE REVIEW

**Comment by jpvajda** [comment]
it looks like the support label job failed on this PR 

```
Run gh pr edit 192 --add-label "support-level:newrelic"
GraphQL error: Resource not accessible by integration
Error: Process completed with exit code 1.
```

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -46,23 +47,23 @@ your interactions with the project and its participants.
 ## Observability Pack Guidelines
 
 We encourage all contributors to actively engage in the creation and maintenance of
-Observability Packs. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
+Quick starts. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
```
Comment (LizBaker) [inline CONTRIBUTING.md:50]: based on the last thread i read (which was a minute ago) i _think_ we're going with lowercase `quickstarts` unless it's the beginning of a sentence, does that sound right to you?

Comment (aswanson-nr) [reply CONTRIBUTING.md:50]: That is correct!

Comment (jpvajda) [reply CONTRIBUTING.md:50]: I knew I forget this at least a dozen times before it sunk in. 🤣

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -46,23 +47,23 @@ your interactions with the project and its participants.
 ## Observability Pack Guidelines
 
 We encourage all contributors to actively engage in the creation and maintenance of
-Observability Packs. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
+quick starts. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
```
Comment (aswanson-nr) [inline CONTRIBUTING.md:50]: I think @nr-kkenney mentioned this in slack, but this should be `quickstarts` not `quick starts`

**Review by aswanson-nr — COMMENTED** [review]
Looks good! Just noticed that this currently refers to packs as `quick starts`, but I think we want it to be `quickstarts`.

File: CONTRIBUTING.md (hunk 2 above)
Comment (zstix) [reply CONTRIBUTING.md:50]:
```suggestion
quickstarts. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!
```

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**